src/Sylius/Bundle/CoreBundle/Fixture/CurrencyFixture.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Fixture;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Sylius\Bundle\FixturesBundle\Fixture\AbstractFixture;
+use Sylius\Component\Currency\Model\CurrencyInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+final class CurrencyFixture extends AbstractFixture
+{
+    /**
+     * @var FactoryInterface
+     */
+    private $currencyFactory;
+
+    /**
+     * @var ObjectManager
+     */
+    private $currencyManager;
+
+    /**
+     * @var string
+     */
+    private $baseCurrencyCode;
+
+    /**
+     * @param FactoryInterface $currencyFactory
+     * @param ObjectManager $currencyManager
+     * @param string $baseCurrencyCode
+     */
+    public function __construct(FactoryInterface $currencyFactory, ObjectManager $currencyManager, $baseCurrencyCode)
```

Comment:
Is the ObjectManager really a currency manager? would call it simply $objectManager